### PR TITLE
Fix typo.

### PR DESCRIPTION
**Pull request by TeoTwawki**

### CODE REVIEW

File: scripts/globals/quests.lua [hunk 1]
```diff
@@ -560,8 +560,8 @@ TRIAL_SIZE_TRIAL_BY_FIRE        = 15; -- + --
 -- Voidwatch (100-105)
 VOIDWATCH_OPS_BORDER_CROSSING   = 100;
 VW_OP_054_ELSHIMO_LIST          = 101;
-WV_OP_101_DETOUR_TO_ZEPWELL     = 102;
-WV_OP_115_LI_TELOR_VARIANT      = 103;
+VW_P_101_DETOUR_TO_ZEPWELL     = 102;
+VW_P_115_LI_TELOR_VARIANT      = 103;
```
Comment (xdemolish) [inline scripts/globals/quests.lua]: VW_P or VW_OP (cause line above is OP)

**Comment by TeoTwawki** [comment]
Putting typos on my typos so I can fix my fix.

Sigh. Would help if I weren't going blind.

**Comment by TeoTwawki** [comment]
@xdemolish thanks for spotting that

**Comment by TeoTwawki** [comment]
corrected now.

**Pull request merged**